management/src/main/resources/reference.conf

```diff
@@ -16,6 +16,45 @@ akka.management {
     # The value will need to be from 0 to 65535.
     port = 19999
 
+    # Use this setting to bind a network interface to a different hostname or ip
+    # than the HTTP Server for Http Cluster Management.
+    # Use "0.0.0.0" to bind to all interfaces.
+    # akka.management.http.hostname if empty
+    bind-hostname = ""
+
+    # Use this setting to bind a network interface to a different port
+    # than the HTTP Server for Http Cluster Management. This may be used
+    # when running akka nodes in a separated networks (under NATs or docker containers).
+    # Use 0 if you want a random available port. Examples:
+    #
+    # akka.management.http.port = 2552
+    # akka.management.http.bind-port = 2553
+    # Network interface will be bound to the 2553 port, but remoting protocol will
+    # expect messages sent to port 2552.
+    #
+    # akka.management.http.port = 0
+    # akka.management.http.bind-port = 0
+    # Network interface will be bound to a random port, and remoting protocol will
+    # expect messages sent to the bound port.
+    #
+    # akka.management.http.port = 2552
+    # akka.management.http.bind-port = 0
+    # Network interface will be bound to a random port, but remoting protocol will
+    # expect messages sent to port 2552.
+    #
+    # akka.management.http.port = 0
+    # akka.management.http.bind-port = 2553
+    # Network interface will be bound to the 2553 port, and remoting protocol will
+    # expect messages sent to the bound port.
+    #
+    # akka.management.http.port = 2552
+    # akka.management.http.bind-port = ""
+    # Network interface will be bound to the 2552 port, and remoting protocol will
+    # expect messages sent to the bound port.
+    #
+    # akka.management.http.port if empty
+    bind-port = ""
+
```

Comment:
I find the examples a bit confusing -- remoting protocol isn't right in this context right?

Reply:
@longshorej Correct, let me fix it

Reply:
Fixed